hr_experience/security/hr_security.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```

Comment:
> Minor Change
> Add readme like this https://github.com/OCA/maintainer-tools/tree/master/template/module/readme
> Otherwise LGTM

@nikul-serpentcs 
I have made the changes as well as applied your suggesition. Could you please verify it again? and thanks for looking it out.